### PR TITLE
Add CODE_OF_CONDUCT, Contributing and issue templates

### PR DESCRIPTION
**Pull request by dasormeter**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (cocreature) [inline .github/ISSUE_TEMPLATE/bug_report.md]: That makes no sense for `ghc-lib`.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (cocreature) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This also looks somewhat confusing. What would be useful is the operating system and the version. Browser is irrelevant.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (cocreature) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I don’t think screenshots make a lot of sense for `ghc-lib`.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (cocreature) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I would just remove those steps, they don’t seem to apply in any useful way to `ghc-lib`.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+# Contributing to DAML
```
Comment (cocreature) [inline CONTRIBUTING.md]: This looks like you copied the `CONTRIBUTING.md` page from the `daml` repo. This needs to be adapted to `ghc-lib`.

**Comment by shayne-fletcher-da** [comment]
@dasormeter This PR seems to be languishing. Are you inclined to finalize it? How do you recommend we proceed?

**Comment by dasormeter** [comment]
hiya @shayne-fletcher-da happy to close this unless someone wants to take it over -- i was more highlighting the file conventions to follow to make these discoverable in the standard way from github but as @cocreature mentioned it does not particularisation for ghc-lib so it is not github-generic and not daml repo-centric.

not gonna have a chance to look at it for another week or so so can either leave it as is or close if you want to replace with a cleaner cut?

**Comment by shayne-fletcher-da** [comment]
@dasormeter There's no urgency, I don't mind leaving it open if there's any chance you'll circle back to it. Let's take stock of this ticket again in a couple of weeks.

**Comment by neil-da** [comment]
I'm inclined to kill this PR. We don't expect this repo to be heavily used, and if it is, the README pretty much describes everything relevant.

**Pull request closed without merging**